crates/ty_python_semantic/resources/mdtest/protocols.md

```diff
@@ -696,7 +696,8 @@ class Foo(Protocol):
         self.b: int = 128  # TODO: should emit diagnostic
 
     def non_init_method(self) -> None:
-        self.y = 64  # fine
+        # TODO: should be fine
```

Comment:
I copied the comment, but should this be fine?
pyright emits diagnostic for this one:
https://pyright-play.net/?code=GYJw9gtgBALgngBwJYDsDmUkQWEMoAK4MYAxmADYBQNpFAhgM6NQBiYYAFEWCeRQEoAXFShioADyGYUMUeLjTGMEPLEATAKbAoAfV2okMfZ0aaKwAVAC0APigA5MCk0jx7qGYsA6CVAC8UAAsAExiAMRQwKiaah5ewN70AVAArABsEVAAKgDyACK5SgAWYACuFOpQmhBGUOpI9GgoYMpIpHHuCd4ARtKo%2BIEAjCEAHFl5hSXlldW1%2BA1NLW0dcVo6LSgGKEa6EJowpeqm5pY29k4ubh5i3XAp6UFZmiDgINIA2qgAbvQUSOprExGEhmvtZABdKAAIlyPQAVppSPgwDp4AhNFAAAYAGSMLz%2BH0eEKxmBYLXwwNBKHoPQomJIUHoMBUSB6ZRgmKxcFJqNgiC5yhAWOhnXE3VIKQA7GEoJFJkVPKUKlUanVFs1WjB2kA

Reply:
FYI @AlexWaygood